hypothesis-python/tests/cover/test_simple_strings.py

```diff
@@ -60,12 +59,6 @@ def test_finds_single_element_strings():
     assert minimal(text(), bool) == "0"
 
 
-@fails_with(AssertionError)
-@given(binary())
-def test_binary_generates_large_examples(x):
-    assert len(x) <= 20
-
-
```

Comment:
This test was unconditionally passing, because it turns out that our `raises` would itself raise an `AssertionError` to signal that nothing was thrown, and thereby accidentally satisfy itself. 🤡 

We do generate long binary strings occasionally, but not for every run, so I've moved this over into the `quality` suite where it belongs.